lady_luck.js

```diff
@@ -303,10 +305,12 @@ function wcAdd(msg, channel_id) {
 	var guild = msg.guild;
 	if (guild == null) {
 		return msg.reply("this command must be used in a guild.");
+		return;
```

Comment:
Now it `return`s the unused result and then has an extraneous `return` statement right afterward. Two `return` statements in direct succession.

Reply:
Merge messed up I guess